app/controllers/admin/claims_controller.rb

```diff
@@ -2,14 +2,28 @@ class Admin::ClaimsController < Admin::BaseAdminController
   before_action :ensure_service_operator
 
   def index
-    claims = Claim.includes(eligibility: [:claim_school, :current_school]).submitted.order(:submitted_at)
-    csv = TslrClaimsCsv.new(claims)
+    @claims = Claim.includes(eligibility: [:claim_school, :current_school]).awaiting_approval.order(:submitted_at)
+    csv = TslrClaimsCsv.new(@claims)
 
     respond_to do |format|
       format.csv { send_file csv.file, type: "text/csv", filename: "claims.csv" }
+      format.html
     end
   end
 
+  def show
+    @claim = Claim.find(params[:id])
+  end
+
+  def approve
```

Comment:
A most RESTful approach would be to implement this as `ClaimApprovalsController#create`.

Reply:
Ah, nice. I like that approach 👍